live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/storage/connector/PreSignedUrlRequesterIntegrationTest.java

```diff
@@ -92,9 +93,10 @@ class PreSignedUrlRequesterIntegrationTest
     private static final String WIREMOCK_TAG = DockerTags.getWiremockTag();
     private static final String NODE_ID = "some-node-ref";
     private static final String PRE_SIGNED_URL_PATH = "/pre-signed-url";
+    private static final String CONTENT_ID = "CONTENT ID";
```

Comment:
Is this constant used anywhere?

Reply:
It's only used in one place (line 135). At one point I was using it in line 99, but since that's a template definition then it's awkward to reuse it there.